Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -3317,6 +3317,16 @@ local Hash = cCryptoHash.sha1HexString("DataToHash")
 					},
 					Notes = "Returns the weapon that the entity has equipped. Returns an empty cItem if no weapon equipped or not applicable.",
 				},
+        GetEquippedShield =
```

Comment:
You used spaces here instead of tabs which caused the indentation to be malformed here.

Reply:
sorry about this, I will adjust my IDE formater